src/main/menu.ts

```diff
@@ -179,20 +179,28 @@ export function getFileMenu(
 ): Electron.MenuItemConstructorOptions {
   const fileMenuNonMac: Electron.MenuItemConstructorOptions = {
     label: tx('global_menu_file_desktop'),
-    submenu: [
-      {
-        label: tx('menu_settings'),
-        click: () => {
-          mainWindow.send('showSettingsDialog')
+    submenu: (() => {
+      let result = [
+        {
+          label: tx('global_menu_file_quit_desktop'),
```

Comment:
I think this should be called "close window" and also I'm not sure if Ctrl+q is appropriate for it. 
I just have some doubts, I don't know the answer for linux and windows
CC @r10s @WofWca

Reply:
On Linux Ctr+q is correct. Windows I'm also not sure

Reply:
for windows, ctrl-q to exit is no mistake. and alt-f4, of couse

Reply:
It's fine I think (as a Windows user)

Reply:
so then on windows and linux both `ctrl+q` and `ctrl+w` close the window, while on macOS `cmd+w` closes just window and `cmd+q` closes the whole app?

Reply:
So I guess we need a new translation string for "close window"?

Reply:
> So I guess we need a new translation string for "close window"?

no, already exists in `_locales/en.xml` as `close_window`